RELEASING.md

```diff
@@ -184,13 +189,16 @@ Update README.md
 ----------------
 After waiting ~1 day and verifying that the release appears on [Maven
 Central](https://search.maven.org/search?q=g:io.grpc), cherry-pick the commit
-that updated the README into the master branch and go through review process.
+that updated the README into the master branch.
 
-```
+```bash
 $ git checkout -b bump-readme master
 $ git cherry-pick v$MAJOR.$MINOR.$PATCH^
+$ git push
```

Comment:
This won't work, right? You need something more like `git push --set-upstream origin bump-readme`

Reply:
I've got autoSetupRemote set to true on pushes, so it works for me, but good point that it should be setup for the default case.